### ISSUE TITLE
Exception: Required attribute Segmentation@multipleModelMethod is not defined

### ISSUE THREAD

**Issue by fenghong007** [jpmml/jpmml-evaluator#266]
When I use multi label (with two labels),it throws:org.jpmml.evaluator.MissingAttributeException: Required attribute Segmentation@multipleModelMethod is not defined.

Some suggestions?

My code:
``` java
ModelEvaluatorBuilder modelEvaluatorBuilder = new ModelEvaluatorBuilder(pmml);

ModelEvaluatorFactory modelEvaluatorFactory = ModelEvaluatorFactory.newInstance();

modelEvaluatorBuilder.setModelEvaluatorFactory(modelEvaluatorFactory);

Evaluator modelEvaluator = modelEvaluatorBuilder.build();
```

**Comment by vruusmann** [comment]
> When I use multi label (with two labels),it throws:org.jpmml.evaluator.MissingAttributeException: Required attribute Segmentation@multipleModelMethod is not defined.

If you open this PMML document in a text editor, then you will see that the `Segmentation@multipleModelMethod` attribute IS DEFINED. Specifically, it's value is `x-multiModelChain`.

This multiple model method is a JPMML vendor extension. It was first released (both into pmml converter and evaluator product verticals) in September 2022:
https://github.com/jpmml/jpmml-evaluator/commit/76afe97ca53cdf37eefceece0b17e9f3ba5ceed4

TLDR: If you're seeing this exception, it means that you're using an outdated JPMML-Evaluator version (that is not aware of this `x-multiModelChain` vendor extension).

The fix is straightforward - upgrade to JPMML-Evaluator version 1.6.4.

> My code:

Your code is un-necessarily complicated.

The following will be just as good:
```java
Evaluator evaluator = new ModelEvaluatorBuilder(pmml)
  .build();
```

**Comment by fenghong007** [comment]
> 

After I updated the version:
pmml-evaluator:1.5.16-->1.6.4
pmml-model:1.5.16-->1.6.4

It works.Thanks a lot.

**Comment by vruusmann** [comment]
> After I updated the version:
> pmml-evaluator:1.5.16-->1.6.4
> pmml-model:1.5.16-->1.6.4
>
> It works.Thanks a lot.
>

There is NO NEED to declare an `org.jpmml:pmml-model` dependency anywhere in your application's build configuration. **The `org.jpmml:pmml-evaluator` dependency alone is enough**.

If you include redundant dependencies, then you're risking with version mismatch (for example, it is not guaranteed that JPMML-Model and JPMML-Evaluator will always be using the same version number at all times - it holds true in the 1.6.X development branch so far, but could "break" any moment).
